FoodplannerApi/Controller/UsersController.cs

```diff
@@ -100,13 +103,73 @@ public async Task<IActionResult> CreateUserChildren([FromBody] UserCreateChildDT
     public async Task<IActionResult> Login([FromBody] Login user)
     {
+
         if (!ModelState.IsValid)
         {
             return BadRequest(ModelState);
         }
         try
         {
             var result = await _userService.GetJWTByEmailAndPasswordAsync(user.Email, user.Password);
+            
+            //TODO: Handle usecase for parent using one time password
+            /*if (!string.IsNullOrEmpty(user.Code) && result != null)
+            {
+                if (Enum.TryParse<UserRole>(result.Role, true, out var parsedRole) && parsedRole == UserRole.Child)
+                {
+                    return BadRequest(new ErrorResponse { Message = ["Børn må ikke laves med dette endpoint, istedet skal LoginChild bruges."] });
+                }
+                var code = await _oneTimePasswordService.GetOneTimePassword(user.Code);
+                code.Used = true;
+                code.UsedByUser = int.Parse(_authService.RetrieveIdFromJwtTokenNoBearer(result.JWT));
+
+                if (await _oneTimePasswordService.UpdateOneTimePassword(code) == 0)
+                    return BadRequest(new ErrorResponse { Message = ["Fejlede i at opdatere engangskode"] });
+                if (await _oneTimePasswordService.RedeemOneTimePassword(code.Code) == 0)
+                    return BadRequest(new ErrorResponse{ Message = ["Fejlede i at indløse engangskode"] });
+            }*/
+
+            if (result != null)
+            {
+                return Ok(result);
+            }
+            return BadRequest(new ErrorResponse { Message = ["Email eller password er forkert"] });
+        }
+        catch (InvalidOperationException e)
+        {
+            return BadRequest(new ErrorResponse { Message = ["Email eller password er forkert"] });
+        }
+    }
+
+    [HttpPost]
+    [ProducesResponseType(typeof(string), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> LoginChild([FromBody] LoginChild user)
+    {
+        if (!ModelState.IsValid)
+        {
+            return BadRequest(ModelState);
+        }
+        try
+        {
+            var result = await _userService.GetJWTByEmailAsync(user.Email);
+            if (!string.IsNullOrEmpty(user.Code) && result != null)
+            {
+                
+
+                var code = await _oneTimePasswordService.GetOneTimePassword(user.Code);
+                if (code.ChildUser != null)
+                    return BadRequest("Only parents can redeem this code.");
+                
+                code.Used = true;
+                code.UsedByUser = int.Parse(_authService.RetrieveIdFromJwtTokenNoBearer(result.JWT));
+                if (await _oneTimePasswordService.UpdateOneTimePassword(code) == 0)
+                    return BadRequest(new ErrorResponse { Message = ["Fejlede i at opdatere engangskode"] });
```

Comment:
Should be like the other return BadRequest and in english:
```suggestion
                    return BadRequest("Failed while updating the one time code");
```